components/ResultsPage/YourAnswers.tsx

```diff
@@ -23,12 +23,20 @@ export const YourAnswers: React.VFC<{
    *    Returns  True   for any other scenario
    */
   function shouldDisplay(input: FieldInput): boolean {
-    const dontShow: String[] = [
+    let returnVal: boolean
+    const exceptions: String[] = [
       'incomeAvailable',
       'partnerIncomeAvailable',
       'oasDefer',
     ]
-    return !dontShow.includes(input.key) && input.value !== 'true'
+
+    !exceptions.includes(input.key)
```

Comment:
There's probably a cleaner way to write this. Nested tertiary statements are hard to read. But let's merge for now to see if everything is working as it should